community/github/pull-requests.md

```diff
@@ -0,0 +1,211 @@
+# 2. pyOpenSci pull request process
+
+When possible, pull request and issue submissions and reviews should follow standard open source workflows. Below are guidelines for pull requests.
+
+## Pull Requests
+
+New pull requests should:
+
+* be created from a fork rather than the pyOpenSci owned parent repository.
+* never be submitted from the `main` branch of your fork.
+
+Authors of new pull requests should whenever possible, do their best to create clean pull requests. 
+
+A clean pull request:
+
+* is focused, and easy to review,
+* can be more quickly reviewed (and in turn merged) so it saves everyone time.
+* closes an existing issue (with the exception of new content such as blog posts that we manage internally using Asana)
+
+:::{tip}
+Review your own pull request before asking someone else to review it for you. You might be surprised that you notice things in the pull request that you didn't notice when working on the content locally.
+:::
+
+### Pull request content 
+
+Below are some guidelines for pull request content. Clean pull requests lead to simpler reviews & faster merging! 
+
+1. **Keep It Small**: Aim for one logical change per pull request to simplify review.
+1. **Create descriptive PR Title and Summary**: Clearly state what the PR achieves and why, including any related issue numbers or discussions.
+1. **Double check that the files committed to the pull request clearly align with the suggested changes being made in the pull request**. For example if a new guidebook page contains images, all images that are new to the guidebook should be included in the list of files in the pr. If the new page reuses images, then link to the existing images in the repository rather than re-adding them.
```

Comment:
```suggestion
1. **Double check that the files committed to the pull request clearly align with the suggested changes being made in the pull request**. For example, if a new guidebook page contains images, all images that are new to the guidebook should be included in the list of files in the pr. If the new page reuses images, then link to the existing images in the repository rather than re-adding them.
```

Reply:
pr should be PR throughout the document